Tests/StreamChatUITests/Mocks/ImageLoader_Mock.swift

```diff
@@ -23,7 +23,12 @@ final class ImageLoader_Mock: ImageLoading {
         with request: ImageDownloadRequest,
         completion: @escaping ((Result<UIImage, Error>) -> Void)
     ) -> Cancellable? {
-        let image = UIImage(data: try! Data(contentsOf: request.url))!
+        var image = UIImage(data: try! Data(contentsOf: request.url))!
+        
+        if let resize = request.options.resize {
+            let cgSize = CGSize(width: resize.width, height: resize.height)
+            image = NukeImageProcessor().scale(image: image, to: cgSize)
+        }
```

Comment:
Our mock ignored the resize option so it work differently in tests and therefore snapshots failed since my fix removed one of the scale cycles in the ChatChanneAvatarView (which we did not need since the NukeImageLoader took care of scaling)